app/admin/accounts/list/route.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return this.store.ajax({
+      url: '/api/v1.0/accounts'
+    }).catch(error => {
+      // handle not found error gracefully
```

Comment:
I don't think we need custom error handling here. Unexpected errors should be handled by error substates.